RELEASE.md

```diff
@@ -0,0 +1,29 @@
+Release 1.2.1 of WebRTC C SDK
+
+Release tagged at: <Commit:  <to-be-filled>
+
+Whats’s new:
+- ICE server stats added
+- RTP outbound and inbound stats added (open source contribution)
+- Signaling stats available
+- Clang check added to Travis
+- MBedTLS support added. Choice between OpenSSL and mbedTLS available
+
+Improvements:
+- ICE candidate priority calculation bug fixed improving the candidate nomination process
+- Improved SDP conformance with RFC to work with CRLF and LF
+- Fixed interruption of sending process due to momentary network unreachable errors (open source contribution)
+
+Producer C Changes: (Commit: e7d4868)
+- Build system improvements
+- README improvements on SDK versioning, development and release
+- Added non-BSD strnstr implementation
+- Fixed static libcurl build failure
+
+Known Issues:
+* Incorrect transceiver direction breaks SDP exchange in Firefox and iOS
+* Reduce memory allocations in sending audio/video packets
```

Comment:
Lets re-phrase it from customer POV.

Reply:
Probably reword this to `High memory allocations in sending audio/video packets`

Reply:
I would phrase as "optimized memory allocations during RTP transmission"